docs/setup.md

```diff
@@ -7,19 +7,21 @@
 1. Install npm package:
 
 ```js
-yarn add @nuxtjs/pwa //or npm i @nuxtjs/pwa
+yarn add --dev @nuxtjs/pwa //or npm i --save-dev @nuxtjs/pwa
 ```
 
 2. Edit your `nuxt.config.js` file to add pwa module:
 
 ```js
 {
-    modules: [
+    buildModules: [
         '@nuxtjs/pwa',
     ],
 }
 ```
 
+ℹ️ If you are using `nuxt < 2.9.0`, use `modules` property instead.
```

Comment:
Can you please mention or `SPA` mode with `nuxt start`?

Reply:
OK I added it. I might not have understood, but I tested with a new nuxt project and the spa mode seems to work fine with nuxt start, both with `modules` and `buildModules`.

As I ran my tests, I realized that I had made a mistake by putting `@nuxtjs/pwa` in the wrong place in the `modules` array, so it works just fine actually. Sorry about that. Is is still relevant to change to the `buildModules` ?

Reply:
Indeed @jsulpis it makes sense to put in devDependencies and buildModules :) Thanks!